.github/scripts/dump_prod_data.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Connect to prod db and copy file to local directory
+aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID
+aws configure set aws_secret_access_key $AWS_SECRET_ACCESS_KEY
+aws configure set default.region "us-east-1"
+
+PROD_SESSION=$(AWS_PAGER="" aws sts assume-role --region us-east-1 --role-arn $PROD_ASSUMED_ROLE_ARN --role-session-name terraform --region us-east-1 --output json)
+unset AWS_PROFILE
+
+unset AWS_ACCESS_KEY_ID
+export AWS_ACCESS_KEY_ID=$(echo $PROD_SESSION | jq -r ".Credentials.AccessKeyId")
+
+unset AWS_SECRET_ACCESS_KEY
+export AWS_SECRET_ACCESS_KEY=$(echo $PROD_SESSION | jq -r ".Credentials.SecretAccessKey")
+
+unset AWS_SESSION_TOKEN
+export AWS_SESSION_TOKEN=$(echo $PROD_SESSION | jq -r ".Credentials.SessionToken")
+
+PROD_KEY=$( aws secretsmanager get-secret-value --secret-id $PROD_DB_KEY_ARN | jq --raw-output '.SecretString' )
+PROD_DB_PASSWORD=$( aws secretsmanager get-secret-value --secret-id $PROD_DB_CREDS_ARN  | jq --raw-output '.SecretString' | jq -r .password )
+PROD_DB_USERNAME=$( aws secretsmanager get-secret-value --secret-id $PROD_DB_CREDS_ARN  | jq --raw-output '.SecretString' | jq -r .username )
+
```

Comment:
I tried for a while to make a "configure_aws" shell script because this chunk of code get used a lot in all the database workflows, but I wasn't able to figure out how to access these three variables (key, password, username) in the parent shell where this script would get called... let me know if there is a way to do that.

Reply:
I think if you call the child script with a `.` in front of it the ENV will get reused from the parent script, I'm not sure if it will work in a github runner though, locally this works

test.sh
```
echo $TEST
. ./test2.sh
echo $TEST2
```

test2.sh
```
echo $TEST
TEST2="TEST2"
```

`TEST=test ./test.sh`
should print
```
test
test
test2
```

but if you remove the first `.` from `. ./test2.sh` then it will only print
```
test
test

```